### PR TITLE
Efficiency Change

### PR DESCRIPTION
**Pull request by maxim-kudryashov**

### CODE REVIEW

File: src/main/java/frc/robot/classes/Kinematics.java [hunk 1]
```diff
@@ -107,8 +101,7 @@ public boolean atPose(Position2D checkpoint, double threshold) {
     double distanceFromPose = distanceFromPose(checkpoint);
     if (distanceFromPose <= threshold) {
       return true;
-    } else {
-      return false;
     }
+    return false;
```
Comment (bjmcternan) [inline src/main/java/frc/robot/classes/Kinematics.java:105]: Let's only have one return statement. 
`boolean retVal = false;`

`blah`
`return retval;`

File: src/main/java/frc/robot/classes/Kinematics.java [hunk 2]
```diff
@@ -81,14 +81,8 @@ public void setPose(Position2D pose) {
...
-
-    while (retval > Math.PI) {
-      retval -= 2 * Math.PI;
-    }
-
-    while (retval < -Math.PI) {
-      retval += 2 * Math.PI;
-    }
+    
+    retval = retval % Math.PI
```
Comment (bjmcternan) [inline src/main/java/frc/robot/classes/Kinematics.java:85]: Missing a `;`

Comment (bjmcternan) [inline src/main/java/frc/robot/classes/Kinematics.java:85]: This is not equivalent to the previous code and will result in an error if the value given is between 1 PI and 2PI.
If you want to simplify this you will need to do `retval = retval % (2*Math.PI);`

**Review by bjmcternan — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/classes/Kinematics.java (hunk 2 above)
Comment (maxim-kudryashov) [reply src/main/java/frc/robot/classes/Kinematics.java:85]: Oh mb. I forgot to add the *2 part. Gj noticing that tho.

**Pull request closed without merging**